presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -163,14 +163,20 @@ public static DoubleWriteFunction doubleWriteFunction()
         return PreparedStatement::setDouble;
     }
 
+    public static ColumnMapping decimalColumnMapping(DecimalType decimalType)
+    {
+        return decimalColumnMapping(decimalType, UNNECESSARY);
+    }
+
     public static ColumnMapping decimalColumnMapping(DecimalType decimalType, RoundingMode roundingMode)
     {
         // JDBC driver can return BigDecimal with lower scale than column's scale when there are trailing zeroes
         int scale = decimalType.getScale();
         if (decimalType.isShort()) {
             return ColumnMapping.longMapping(
                     decimalType,
-                    shortDecimalReadFunction(decimalType, UNNECESSARY),
+                    // TODO why roundingMode is not used here, but it is used for long decimal only?
```

Comment:
this TODO will not drive any actions. If you want to make it actionable create an issue for that; tag in comment; and assign suspected owner.

Reply:
@losipiuk  let's continue above -- https://github.com/prestosql/presto/pull/6026#discussion_r527765461

Reply:
Yeah sure. My point was just that, if we merge this one, without conclusion, the TODO will likely be forgotten.